values/tags.yaml

```diff
@@ -10,5 +10,5 @@ repositories:
 registry:
   url: consensys-docker-pegasys-orchestrate.bintray.io
   credentials:
-    username: <ACCOUNT_LOGIN>
-    password: <ACCOUNT_PASSWORD>
+    username: covantis@consensys
+    password: fe67fd2465e8a98b9ce27f567e8d92956b202ec5
```

Comment:
What was the reason behind this change? If this credentials are valid it would be better to suggest the users to introduce them instead of having them here as it used to be

Reply:
a miss from my part, I have to remove them